model/process/registry.yaml

```
@@ -11,6 +11,14 @@ groups:
         brief: >
           Process identifier (PID).
         examples: [1234]
+      - id: process.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the process. On Linux based systems, can be set
+          to the `Name` in `proc/[pid]/status`. On Windows, can be set to the
```

```suggestion
          to the value in `/proc/[pid]/comm`. On Windows, can be set to the
```

We can also use `/proc/[pid]/comm` which requires no parsing, unlike extracting `Name` out of `/proc/[pid]/status`. The values should be equivalent.

Also see:

- [proc_pid_comm(5)](https://man7.org/linux/man-pages/man5/proc_pid_comm.5.html)
- [prod_pid_status(5)](https://man7.org/linux/man-pages/man5/proc_pid_status.5.html)

---

What do you think about suggesting both since they're the same anyway? For example, in the hostmetricsreceiver we'll already have parsed `/proc/[pid]/status` for other information, so just getting the name from that is fine in our case, but in other cases people might prefer to just get `/proc/[pid]/comm`.

---

Done in [5a679fd](https://github.com/open-telemetry/semantic-conventions/pull/1737/commits/5a679fd2d86cbc3f5b6584f59b37bd5051dd6f79)